### ISSUE TITLE
Shift to using official images as base

### ISSUE THREAD

**Issue by mhart** [lambci/docker-lambda#299]
Now that SAM CLI is v1.0.0, AWS has published some official images, such as `amazon/aws-sam-cli-emulation-image-java11`

These don't seem to be documented anywhere, or open source, but it's probably better in the long run to look into using these instead of the current approach.

**Comment by mnapoli** [comment]
Hi @mhart, over at [bref]() we've been using your images to build PHP runtimes for Lambda. Thanks a lot for that!

I'm wondering whether we should switch to official AWS images, the thing is though I can't find them on Docker: https://hub.docker.com/search?q=amazon%20lambda&type=image The one you mention seems very specific, is there any equivalent to `lambci/lambda` and would you recommend using it?

Thanks a lot!

**Comment by mhart** [comment]
Hey Matthieu, glad they've been useful to you for bref! I remember when I got PHP running many moons ago on Lambda (running [Laravel's test suite](https://lambci-public-buildresults-e3xwlufrwb3i.s3.amazonaws.com/gh/mhart/test-ci-project/builds/140/3e9954f8a74dda5f4050b65bcf56cde8.html) no less 😆 ) – so it's great to see y'all have created a home for it.

In terms of official AWS images – I'd say check in in a month or two 😉

**Comment by mnapoli** [comment]
Right!

And re Laravel tests, that's awesome! Reminds me of a project I started back then: a CI running on Lambda ^^
I managed to compile MySQL and run it inside Lambda, so that functional tests could run in 1000× parallelization but each test runner still has its own independent MySQL instance (and yes the data is lost, but for functional tests that's fine).

**Comment by mhart** [comment]
Yeah, that's exactly what we do – with Postgres, Redis and Elasticsearch. I'll be talking more about this (running GitHub Actions on Lambda specifically) for reInvent 👍

**Comment by mnapoli** [comment]
Oh yeahhh 😏
